internal/backends/qt/qt_widgets/palette.rs

```diff
@@ -41,6 +41,7 @@ pub struct NativePalette {
     pub selection_background: Property<Brush>,
     pub selection_foreground: Property<Brush>,
     pub border: Property<Brush>,
+    pub dark_color_scheme: Property<bool>, // This is a no-op as Qt doesn't allow changing the palette yet.
```

Comment:
This should then use the `StyleMetrics.dark-color-scheme`  which is initialized depending on the Palette's color

Reply:
Ah yes, good point. Too bad I can't access the instances :(. I guess I have to duplicate the code.